rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -1079,21 +1189,27 @@ def _ordered_tag_specs(
         return ordered_tag_spec
 
     def _check_data(self) -> None:
-        if TEXT_FEATURES not in self.data_signature:
+        if (
+            TEXT_SENTENCE_FEATURES not in self.data_signature
+            and TEXT_SEQUENCE_FEATURES not in self.data_signature
+        ):
             raise InvalidConfigError(
                 f"No text features specified. "
                 f"Cannot train '{self.__class__.__name__}' model."
             )
         if self.config[INTENT_CLASSIFICATION]:
-            if LABEL_FEATURES not in self.data_signature:
+            if (
+                LABEL_SENTENCE_FEATURES not in self.data_signature
+                and LABEL_SEQUENCE_FEATURES not in self.data_signature
+            ):
                 raise InvalidConfigError(
                     f"No label features specified. "
                     f"Cannot train '{self.__class__.__name__}' model."
                 )
             if (
                 self.config[SHARE_HIDDEN_LAYERS]
-                and self.data_signature[TEXT_FEATURES]
-                != self.data_signature[LABEL_FEATURES]
+                and self.data_signature[TEXT_SENTENCE_FEATURES]
+                != self.data_signature[LABEL_SENTENCE_FEATURES]
```

Comment:
why we check only sentence features here?

Reply:
good point, will add a check for sequence features as well